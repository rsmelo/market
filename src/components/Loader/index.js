import React from 'react'

import style from './style.css'

const Loader = () => (
  <div className={style.overlay}>
    <span className={style.loader}>Loading...</span>
  </div>
)

export default Loader
