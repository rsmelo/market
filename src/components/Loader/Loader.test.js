import React from 'react'
import renderer from 'react-test-renderer'

import Loader from './index'

jest.mock('react-router-dom', () => ({ Link: 'Link' }))

describe('Loader', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Loader />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
