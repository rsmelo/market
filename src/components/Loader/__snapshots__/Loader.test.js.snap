// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<div
  className={undefined}
>
  <span
    className={undefined}
  >
    Loading...
  </span>
</div>
`;
